addons/main.go

```diff
@@ -147,6 +152,17 @@ func main() {
 		os.Exit(1)
 	}
 
+	tanzuClusterBootstrapReconciler := controllers.NewtanzuClusterBootstrapReconciler(mgr.GetClient(),
```

Comment:
Do we want to add ClusterBootstrap controller to main.go now?

Reply:
Not yet, just removed.